models/models.go

```diff
@@ -24,6 +24,9 @@ import (
 	// Needed for the MSSSQL driver
 	_ "github.com/denisenkom/go-mssqldb"
 
+	// Needed for SQLite
+	_ "github.com/mattn/go-sqlite3"
```

Comment:
I know, I added it so the tests would work; I realize that sqlite has to be explicitly enable. Perhaps I should have made that clearer.